interface/main/mobile/m_cal.php

```diff
@@ -0,0 +1,390 @@
+        margin:5px;
+        padding:5px 20px;
+        box-shadow: 1px 1px 2px #938282;
+    }
+    label input {
+        padding:left:30px;
+    }
+    .byCatDisplay {
+        display:none;
+    }
+    .btn {
+        font-size: 1.5rem;
+    }
+    .card-title {
+        overflow:hidden;
+    }
+    .jumbotronA {
+        min-height:400px;
+        margin:8px;
+    }
+    td {
+        text-align: center;
+    }
+    @media (min-width:1200px){
+        .auto-clear .col-lg-1:nth-child(12n+1){clear:left;}
+        .auto-clear .col-lg-2:nth-child(6n+1){clear:left;}
+        .auto-clear .col-lg-3:nth-child(4n+1){clear:left;}
+        .auto-clear .col-lg-4:nth-child(3n+1){clear:left;}
+        .auto-clear .col-lg-6:nth-child(odd){clear:left;}
+    }
+    @media (min-width:992px) and (max-width:1199px){
+        .auto-clear .col-md-1:nth-child(12n+1){clear:left;}
+        .auto-clear .col-md-2:nth-child(6n+1){clear:left;}
+        .auto-clear .col-md-3:nth-child(4n+1){clear:left;}
+        .auto-clear .col-md-4:nth-child(3n+1){clear:left;}
+        .auto-clear .col-md-6:nth-child(odd){clear:left;}
+    }
+    @media (min-width:768px) and (max-width:991px){
+        .auto-clear .col-sm-1:nth-child(12n+1){clear:left;}
+        .auto-clear .col-sm-2:nth-child(6n+1){clear:left;}
+        .auto-clear .col-sm-3:nth-child(4n+1){clear:left;}
+        .auto-clear .col-sm-4:nth-child(3n+1){clear:left;}
+        .auto-clear .col-sm-6:nth-child(odd){clear:left;}
+    }
+    @media (max-width:767px) {
+        .auto-clear .col-xs-1:nth-child(12n+1) {clear: left;}
+        .auto-clear .col-xs-2:nth-child(6n+1) {clear: left;}
+        .auto-clear .col-xs-3:nth-child(4n+1) {clear: left;}
+        .auto-clear .col-xs-4:nth-child(3n+1) {clear: left;}
+        .auto-clear .col-xs-6:nth-child(odd) {clear: left;}
+        .jumbotronA {margin: 8px auto;}
+        #head_img {margin: 2vH 0 0 0;max-height: 15vH;}
+    }
+
+    @media (max-width:400px){
+        .auto-clear .col-xs-1:nth-child(12n+1){clear:left;}
+        .auto-clear .col-xs-2:nth-child(6n+1){clear:left;}
+        .auto-clear .col-xs-3:nth-child(4n+1){clear:left;}
+        .auto-clear .col-xs-4:nth-child(3n+1){clear:left;}
+        .auto-clear .col-xs-6:nth-child(odd){clear:left;}
+        .jumbotronA {margin: 8px auto;}
+        #head_img {margin: 2vH 0 0 0;max-height: 10vH;}
+    }
+    .section_title {font-size:1.2em;text-decoration:underline;font-weight:600;margin-bottom:8px;}
+</style>
+<body style="background-color: #fff;" >
+<?php common_header($display); ?>
+
+    <?php
+    if ($_GET['eid']) {
+        echo "<script>$(document).ready(function () { ScrollIt(); }); </script>";
+    }
+        
+        $day        = $_GET['day'];
+        $month      = $_GET['month'];
+        $provider   = $_GET['provider'];
+        $eid        = $_GET['eid'];
+        
+        $day        = text($day);
+        $month      = text($month);
+        $provider   = text($provider);
+        $eid        = text($eid);
```

Comment:
recommend not escaping variables until they are echo'd. Or else it's too difficult to track. And note some places need attr() and some places need text()